src/templates/postTemplate.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import Layout from "../components/layout";
+
+const PostTemplate = ({ data }) => {
```

Comment:
This is the template that blog posts are based off of. It converts the markdown to an html document based off of this template for any blog post that is put on the pages/blogs folder.